storage/elasticsearch/elasticsearch.go

```diff
@@ -47,7 +47,7 @@ func (c *ElasticSearchStorage) StoreFlows(flows []*flow.Flow) error {
 			logging.GetLogger().Debug("Indexing: %s", string(j))
 		}
 
-		_, err = c.connection.Index("skydive", "flow", flow.GetUUID(), nil, *flow)
+		_, err = c.connection.Index("skydive", "flow", flow.UUID, nil, *flow)
```

Comment:
Since the Flow schema changed, maybe we could add something to convert new flow format to the old one in order to keep the index working while thinking to another way of storing. So that we will not have feature regression.